src/config.h

```diff
@@ -157,3 +157,33 @@
 #ifndef MG_ENABLE_PROFILE
 #define MG_ENABLE_PROFILE 0
 #endif
+
+#ifndef MG_ENABLE_DRIVER_INIT
+#define MG_ENABLE_DRIVER_INIT 1
+#endif
+
+#ifndef MG_TCPIP_IP
+#define MG_TCPIP_IP 0
+#endif
+
+#ifndef MG_TCPIP_MASK
+#define MG_TCPIP_MASK 0
+#endif
+
+#ifndef MG_TCPIP_GW
+#define MG_TCPIP_GW 0
+#endif
+
+#define MG_MAC_ADDRESS_RANDOM \
+  { 0, 0, 0, 0, 0, 0 }
+
+#define MG_MAC_ADDRESS_STM32_UID(ADDR)                       \
```

Comment:
I would move MG_MAC_ADDRESS_${DEVICE}_UID into drivers/${DEVICE}.h, since this is dependent on the platform and have it like before (without any arguments).